src/StitchPreview.ts

```diff
@@ -79,8 +79,8 @@ export class StitchPreview extends Disposable implements vscode.Disposable {
         const context = ContextHandler.getContext();
         if (!context) {
             this._handleStitchError({
-                title: `No ${CONSTANTS.integrationExtension} file found`,
-                description: `Please open an *${CONSTANTS.integrationExtension} file or directory to enable the preview!`
+                title: `No ${CONSTANTS.integrationExtensionJson} or ${CONSTANTS.integrationExtensionYaml} file found`,
```

Comment:
Should this also contain the yml extension and then we might need to change the entire message.